announcements/1.14.1.md

```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.14.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+ * Android security updates for June 2022 (see [Android Security Bulletin - June 2022](https://source.android.com/security/bulletin/2022-06-01) for more information)
+ * Android WebView has been updated to [102.0.5005.78](https://chromereleases.googleblog.com/2022/05/chrome-for-android-update_28.html)
+ * The Android [system app installation process](https://discourse.ubuntu.com/t/how-to-install-an-apk-as-a-system-app/27086) now supports the [APK Signature Scheme v2](https://source.android.com/security/apksigning/v2)
+
+#### Bugs
+
+* AC-930 AMS doesn't retry stripping a container when LXD returns not matching ETag
+* AC-936 AMS is misbehaving when LXD cluster failing to process requests
+* AC-929 Node controller doesn't start due to missing route utility
+* AC-928 Use ip over ifconfig command to parse ip address of the network device in Anbox container
```

Comment:
```suggestion
* AC-928 Use the `ip` command rather than `ifconfig` to parse the IP address of the network device in Anbox container
```
If that is what we mean?